.travis.yml

```diff
@@ -13,6 +13,7 @@ matrix:
       script:
         - bundle exec rake
         - bundle exec rubocop
+        - bundle exec bundle-audit update && bundle exec bundle-audit check
```

Comment:
Why are the two `ruby` entries here different `language`s, rather than just being two entries in the `rvm` field of one `ruby` language? All the other steps look the same otherwise.

Reply:
Not sure why, but I agree that 2 entries in the `rvm` field would be better in this case, will update

Reply:
After trying it, it doesn't look like it does 2 build/tests for the 2 different ruby versions if you do it that way.